src/gui/DatabaseWidget.cpp

```diff
@@ -1986,6 +2013,7 @@ bool DatabaseWidget::save()
     if (performSave(errorMessage)) {
         m_saveAttempts = 0;
         m_blockAutoSave = false;
+        m_autosaveTimer->stop(); // user save reset autosave delay - avoid triggering another save
```

Comment:
About comment. This `save` method is not always called by user, it is also called by autosave. I would rephrase it as `reset autosave delay to avoid triggering another save`.

Reply:
Fixed